.github/workflows/ci.yml

```diff
@@ -207,6 +207,7 @@ jobs:
           nimble test
 
   bumpNBC-stable:
+    if: github.ref == 'refs/heads/master'
```

Comment:
We should change to `master` to `stable`.

Reply:
Agreed.

Reply:
I propose we keep master for backward compatibility and just leave it as it is tho, does that work for you @dryajov ?
And of course set stable as default.

Reply:
Altho dunno, ref might be enough, someone pulling master might be better off with error message to know they have to change.